app/src/main/java/com/zulip/android/LoginActivity.java

```diff
@@ -198,6 +198,7 @@ public void onTaskFailure(String result) {
                     // Invalidate the token and try again, unless the user we
                     // are authenticating as is not registered or is disabled.
                     if (!loginTask.userDefinitelyInvalid) {
+                        GoogleAuthUtil.invalidateToken(that, token);
```

Comment:
tbh I'm not sure what the deal is with this commit. This line was removed in https://github.com/zulip/zulip-android/pull/52/commits/6fca42e1be03216dd713f053b66be9aca7ebbe37